tests/lib/libc_api/src/main.c

```diff
@@ -0,0 +1,397 @@
+/*
+ * Copyright (c) 2017 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * @file test access to the minimal C libraries
+ *
+ * This module verifies that the various minimal C libraries can be used.
+ *
+ * IMPORTANT: The module only ensures that each supported library is present,
+ * and that a bare minimum of its functionality is operating correctly. It does
+ * NOT guarantee that ALL standards-defined functionality is present, nor does
+ * it guarantee that ALL functionality provided is working correctly.
+ */
```

Comment:
This was literally copied and pasted from `tests/lib/c_lib/src/main.c`.

If this is indeed meant to be a test for the minimal C library only, there is no need to create a separate test for it. You just need to extend the existing `c_lib` test.

Reply:
Thanks for comments, done.